### PR TITLE
[FIX] Ot198-16-FIX Roles model, migration and controller

### PR DESCRIPTION
**Pull request by penalvapablo**
[Ot198-16-FIX](https://alkemy-labs.atlassian.net/browse/OT198-16)

## Description:
- Delete Roles model and migration
- Create Role model and migration
- Fix Role controller

### Model fields:
id: INTEGER NOT NULL AUTO_INCREMENT
name: VARCHAR NOT NULL
description: VARCHAR NULLABLE
timestamps

## Evidence:

image
image

### CODE REVIEW

**Review by gabodmc — APPROVED** [review]

File: database/models/role.js [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+  class Role extends Model {
     /**
      * Helper method for defining associations.
      * This method is not a part of Sequelize lifecycle.
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
       // define association here
-      Roles.hasMany(models.Users, {
+      Role.hasMany(models.Users, {
```
Comment (honeybadger2788) [inline database/models/role.js:14]: cambiar a model.User

**Review by honeybadger2788 — CHANGES_REQUESTED** [review]

**Review by honeybadger2788 — APPROVED** [review]
Excelente, Pablo!

**Review by JoaquinBar1 — APPROVED** [review]
Todo ok!

**Review by HernanGarcia1985 — APPROVED** [review]
Realizó los cambios que le habían sido solicitados!!!

**Pull request merged**